yarn.lock

```diff
@@ -1660,6 +1654,13 @@ jest-snapshot@^18.0.0:
     natural-compare "^1.4.0"
     pretty-format "^18.0.0"
 
+jest-test-typescript-parser@18.1.0:
+  version "19.0.2"
```

Comment:
lol, looks like you got the latest version of the parser, but package.json doesn't have it.